src/locales/en/common.json

```diff
@@ -104,6 +104,7 @@
   "Caution! You are about to stake for the punished validator, this will result in your LSK tokens being locked for a period of {{daysLeft}} days. In addition, please note that your stake will not be counted until the {{daysLeft}} day period has expired. ": "Caution! You are about to stake for the punished validator, this will result in your LSK tokens being locked for a period of {{daysLeft}} days. In addition, please note that your stake will not be counted until the {{daysLeft}} day period has expired. ",
```

Comment:
Not related to this line (NRTTL)

Most files needs to be prettified.

Reply:
this file is auto generated by i18nscanner if touch it will be change again for each build

Reply:
The comment is not not related to this line nor file. 
Just a global comment for the PR,  that most of the changed files in this PR needs to be prettified.

Reply:
**For example this file:**
<img width="857" alt="image" src="https://user-images.githubusercontent.com/8784876/218090378-46bc2503-f37f-414e-bdfb-bb2ec346330f.png">

**After i prettify it, it looks like this:**
<img width="872" alt="image" src="https://user-images.githubusercontent.com/8784876/218090527-6a4e54f9-8517-4287-b25f-516ddfd5139d.png">